src/com/nicoles_change_maker/ChangeMaker.java

```diff
@@ -0,0 +1,61 @@
+package com.nicoles_change_maker;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Iterator;
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class ChangeMaker {
+    double remainingDollars;
+
+    public HashMap makeChange(double dollarAmount) {
```

Comment:
Floats and doubles are imprecise, and it's a good practice not to use them to represent money: https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency/3730249#3730249. 

(Same thing in Ruby: use `Integer` or `BigDecimal` instead).